### PR TITLE
chore(skip-release): Prepare 1.9.0

### LINKED ISSUE #784: Prepare for 1.9.0 release
see https://github.com/redhat-developer/intellij-openshift-connector/milestone/41

### PR DESCRIPTION
**Pull request by sbouchet**
## Was the change discussed in an issue?
fixes #784

### CODE REVIEW

File: src/main/resources/META-INF/plugin.xml [hunk 1]
```diff
@@ -17,6 +17,12 @@
     ]]></description>
 
   <change-notes><![CDATA[
+  <p><b>1.9.0</b></p>
+  <ul>
+  <li>Support IDEA 2024.1</li>
+  <li>Minimal supported IDEA version bumped to 2022.3</li>
+  <li>Bug fixes</li>
```
Comment (mohitsuman) [inline src/main/resources/META-INF/plugin.xml:24]:
```suggestion
  <li>Improvements around `start` and `stop` for OpenShift components</li>
```

**Review by mohitsuman — CHANGES_REQUESTED** [review]
wording changes

**Review by adietish — APPROVED** [review]
lgtm

**Pull request merged**